### PR TITLE
creating a build for mat-editor

### PR DESCRIPTION
**Pull request by rajitha1998**
## Changes proposed in the pull request
  * Creating a build check for Mat-editor using Git Hub actions
## Impact
 * nodejs.yml

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# For new features or enhancements
+## To-do (Overall work needs to be done)
```
Comment (Anuranie) [inline .github/ISSUE_TEMPLATE.md]: Better to say 'Overall work to be done'

**Review by psnmissaka — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+## Expected Behavior
```
Comment (Anuranie) [inline .github/ISSUE_TEMPLATE.md]: 'Behaviour'

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# For new features or enhancements
+## To-do (Overall work needs to be done)
+## Expected Behavior 
+## Impact
+## Other 
+
+
+# For bugs
+## Current Behavior
```
Comment (Anuranie) [inline .github/ISSUE_TEMPLATE.md]: 'Behaviour'

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+<!--- what happens instead of the expected behavior -->
```
Comment (Anuranie) [inline .github/ISSUE_TEMPLATE.md]: 'What' and 'behaviour'

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+## Expected Behavior
```
Comment (Anuranie) [inline .github/ISSUE_TEMPLATE.md]: 'Behaviour'

File: .github/ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+<!--- what should happen -->
```
Comment (Anuranie) [inline .github/ISSUE_TEMPLATE.md]: 'What'

File: .github/ISSUE_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+## Steps to Reproduce
+1.
+2.
+## Identified Cause
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
```
Comment (Anuranie) [inline .github/ISSUE_TEMPLATE.md]: remove the comma at the end

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
+### Fixes #XXXX
+## Changes proposed in the pull request
+  *
+## Impact
+## Other information
+
+Check if you have done the following before creating the PR
+- [ ] console logs have been deleted
```
Comment (Anuranie) [inline .github/PULL_REQUEST_TEMPLATE.md]: 'Console'

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+- [ ] lint issues are checked and fixed with `ng lint`
```
Comment (Anuranie) [inline .github/PULL_REQUEST_TEMPLATE.md]: 'Lint'

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+- [ ] Tested for a build (ran `ng build mat-editor` without errors)
+- [ ] No commented code (If commented the reasons are given)
```
Comment (Anuranie) [inline .github/PULL_REQUEST_TEMPLATE.md]: 'if'

**Review by Anuranie — APPROVED** [review]

**Pull request merged**